src/github_review_prompts/github_client.py

```
@@ -0,0 +1,706 @@
+                    error_msg += " レート制限に達している可能性があります。"
+                else:
+                    error_msg += " トークンの権限を確認してください。"
+                raise APIError(error_msg, status_code=403)
+            
+            # その他のHTTPエラー
+            if not response.ok:
+                error_data = {}
+                try:
+                    error_data = response.json()
+                except (ValueError, requests.exceptions.JSONDecodeError):
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(ValueError, requests.exceptions.JSONDecodeError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ValueError, requests.exceptions.JSONDecodeError)`

(SIM105)